### ISSUE TITLE
$formatDatabaseJson doesn't work for a newer versions of this library

### ISSUE THREAD

**Issue by nasushkov** [Vincit/objection-graphql#17]
Hi
I'm trying to upgrade from version 0.1.2. I use _snake_case_  for db objects naming and _camelCase_ for my ORM models. I use the following code to parse from one format to another:

```
  $formatDatabaseJson(json) {
        json = super.$formatDatabaseJson(json)
        return mapKeys(json, (value, key) => snakeCase(key))
    }
    $parseDatabaseJson(json) {
        json = mapKeys(json, (value, key) => camelCase(key))
        return super.$parseDatabaseJson(json)
    }
```

My queries used to work for version 0.1.2, but now they fail  and I think the reason is wrong parsing behavior as I've got wrong queries in my log:

> select "area_param"."paramName", "area_param"."type", "area_param"."displayUnit", "area_param"."displayName" from "area_param"

**Comment by koskimas** [comment]
Thanks for reporting this. I'll check out what is going on.

**Comment by koskimas** [comment]
Could you put this in your package.json

```
"objection": "git://github.com/Vincit/objection-graphql.git#382d2ed"
```

and test if it fixed you bug?

**Comment by nasushkov** [comment]
@koskimas Thanks! I'll try it

**Comment by nasushkov** [comment]
@koskimas it works!

**Comment by koskimas** [comment]
Great! I'll write couple of test cases to prevent this from happening again and I'll publish a patch after that.

**Comment by koskimas** [comment]
Fixed in v0.2.3
